ingestion-beam/src/main/java/com/mozilla/telemetry/republisher/RepublishPerDocTypeEnabled.java

```diff
@@ -0,0 +1,100 @@
+package com.mozilla.telemetry.republisher;
+
+import com.mozilla.telemetry.transforms.PubsubConstraints;
+import java.io.Serializable;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.apache.beam.sdk.io.gcp.pubsub.PubsubMessage;
+import org.apache.beam.sdk.options.ValueProvider.StaticValueProvider;
+import org.apache.beam.sdk.transforms.PTransform;
+import org.apache.beam.sdk.transforms.Partition;
+import org.apache.beam.sdk.values.PCollection;
+import org.apache.beam.sdk.values.PCollectionList;
+import org.apache.beam.sdk.values.PDone;
+
+// Handles deprecated --perDocTypeDestination and --perDocTypeEnabledList options
+public class RepublishPerDocTypeEnabled extends PTransform<PCollection<PubsubMessage>, PDone> {
```

Comment:
This class is the old `RepublishPerDocType`, I just renamed it for backwards compatibility. Once we decide to remove the `--perDocTypeDestination` and `--perDocTypeEnabledList` options entirely this class can simply be removed.